components/Autocomplete.vue

```diff
@@ -159,6 +164,28 @@
   color: white;
 }
 
+
+/*Style for border round the search options*/
+.search_op {
+  border: 1px solid #61ae25;
+  border-radius: 0.2rem;
+}
+
+div .ref-div:focus {
+  background-color: #fff;
+  border-color: #80bdff;
+  outline: 0;
+  border-radius: 0.2rem;
+  box-shadow: 0 0 0 0.2rem rgba(0, 123, 255, 0.25);
+}
+
+.focus-class:focus {
+  border-color: #ced4da;
+  border-right: none;
+  -webkit-box-shadow: none;
```

Comment:
Is this needed? I was under the impression autoprefixer was working out of the box with next.js, if not I would recommend we have a task to add it. Adding prefixes will become tedious.